🌐 Add French translation for `docs/fr/docs/tutorial/tests.md`

Mgcsuper opened the pull request:
A possible translation in frensh of the doc file testing.md

Mgcsuper committed:
Add files via upload

A possible translation in frensh of the doc file testing.md

pre-commit-ci[bot] committed:
🎨 [pre-commit.ci] Auto format from pre-commit.com hooks

tiangolo commented:
📝 Docs preview for commit 8d6b290d0943c286f72555bc2815577c3ff37edf at: https://a0d7986f.fastapitiangolo.pages.dev

alejsdev committed:
Merge branch 'master' into master

tiangolo commented:
📝 Docs preview for commit a171dbd3f50465984c7b221a012f8f3d359f8ac4 at: https://f9b182bf.fastapitiangolo.pages.dev

Mgcsuper committed:
Merge branch 'tiangolo:master' into master

Mgcsuper committed:
Create fastapi/tests/test_tutorial/test_path_params /test_tutorial003.py

It can be interesting to note that although we have created two queries with the same path there are still two roots, one does not overwrite the other

pre-commit-ci[bot] committed:
🎨 [pre-commit.ci] Auto format from pre-commit.com hooks

Mgcsuper committed:
Update and rename test_tutorial003.py to test_tutorial003b.py

it was tutorial003b with over fonction names

pre-commit-ci[bot] committed:
🎨 [pre-commit.ci] Auto format from pre-commit.com hooks

Mgcsuper commented:
I don't know how to do two different pull requests

tiangolo commented:
📝 Docs preview for commit d49ee02f440787d064c6fdddf3468512c92ffcd0 at: https://cde026da.fastapitiangolo.pages.dev

alejsdev committed:
Merge branch 'master' into master

tiangolo commented:
📝 Docs preview for commit ab36385ded0135479d421010c96a2a06ab217d44 at: https://962b54f3.fastapitiangolo.pages.dev

Mgcsuper committed:
Merge branch 'tiangolo:master' into master

tiangolo commented:
📝 Docs preview for commit 04ec885accdf00f6f820dae9a1d45c2ec8b9eda4 at: https://1f7cc2c4.fastapitiangolo.pages.dev

alejsdev committed:
Merge branch 'master' into master

tiangolo commented:
📝 Docs preview for commit dab148cba8f4b093174d66c2b0057943810a6d4a at: https://21a82c31.fastapitiangolo.pages.dev

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+
+Utilisez l'objet `TestClient` de la même manière que vous le feriez avec `httpx`.
+
+Écrivez des instruction d'assertion `assert` simples avec les expressions Python standard qu'il est necessaire de vérifier (de nouveau, c'est une pratique standard de `pytest`)."
```
```suggestion
Écrivez des instructions d'assertion `assert` simples avec les expressions Python standard qu'il est necessaire de vérifier (de nouveau, c'est une pratique standard de `pytest`)."
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
````diff
@@ -0,0 +1,216 @@
+```
+
+!!! astuce
+    il est utile de remarquer que les fonction de test sont normal `def`, et non `async def`
````
```suggestion
    il est important de remarquer que les fonction de tests sont normal `def`, et non `async def`
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+!!! astuce
+    il est utile de remarquer que les fonction de test sont normal `def`, et non `async def`
+
+    Et les appel au client sont aussi normaux, pas besoin d'utiliser `await`
```
```suggestion
    Les appels au client sont aussi normaux, pas besoin d'utiliser `await`
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+
+
+!!! astuce
+    Si vous chercher tout de même à utiliser la foncion `async` dans vos test, à l'exeption d’envoyer des requêtes à votre application FastAPI (par exemple, des fonctions de base de données asynchrones), jetez un coup d’œil aux [Tests asynchrones (en)](../advanced/async-tests.md) {.internal-link target=_blank} dans le tutoriel avancé.
```
```suggestion
        Si vous devez utiliser `async` dans vos tests (par exemple, pour des fonctions de base de données asynchrones), sauf pour envoyer des requêtes à votre application FastAPI, consultez les [Tests asynchrones](../advanced/async-tests.md) {.internal-link target=_blank} dans le tutoriel avancé.
.

```

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+
+Dans une application réelle, vous auriez probablement vos tests dans un fichier différent.
+
+Et votre application **FastAPI** peut également être composée de plusieurs fichiers/modules, etc.
```
```suggestion
Votre application **FastAPI** peut également être composée de plusieurs fichiers/modules, etc.
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+
+### fichier app de **FastAPI**
+
+disons que vous ayez une structure de fichier comme décrit dans [application de grande taille (en)](./bigger-applications.md) {.internal-link target=_blank}:
```
```suggestion
Disons que vous ayez une structure de fichier comme décrit dans [application de grande taille (en)](./bigger-applications.md) {.internal-link target=_blank}:
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+
+Et votre application **FastAPI** peut également être composée de plusieurs fichiers/modules, etc.
+
+### fichier app de **FastAPI**
```
```suggestion
### Fchier app de **FastAPI**
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
````diff
@@ -0,0 +1,216 @@
+│   └── main.py
+```
+
+dans le fichier `main.py` vous avez votre application **FastAPI** :
````
```suggestion
Le fichier `main.py` vous permet d'accéder à votre application **FastAPI** :
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
````diff
@@ -0,0 +1,216 @@
+{!../../../docs_src/app_testing/main.py!}
+```
+
+### fichier de test
````
```suggestion
### Fichier de test
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+
+## Test: exemple supplémentaire
+
+Allons plus loin dans cet exemple et ajoutons plus de détails pour voir comment tester différentes pièces.
```
```suggestion
Pour aller plus loin dans cet exemple, ajoutons plus de détails pour observer comment tester différentes pièces.
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+
+Allons plus loin dans cet exemple et ajoutons plus de détails pour voir comment tester différentes pièces.
+
+### fichier app étendu de **FastAPI**
```
```suggestion
### Fichier App étendu de **FastAPI**
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
````diff
@@ -0,0 +1,216 @@
+│   └── test_main.py
+```
+
+Supposons que maintenant le fichier `main.py` avec votre application **FastAPI** a d’autres **opérations sur chemin d'accès**.
````
```suggestion
Supposons que maintenant le fichier `main.py` avec votre application **FastAPI** à d’autres **opérations sur chemin d'accès**.
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+
+Il dispose d’une opération `GET` qui pourrait renvoyer une erreur.
+
+Et dispose d’une opération `POST` qui peut renvoyer plusieurs erreurs.
```
```suggestion
Il  dispose d’une opération `POST` qui peut renvoyer plusieurs erreurs.
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
````diff
@@ -0,0 +1,216 @@
+{!> ../../../docs_src/app_testing/app_b/main.py!}
+```
+
+### fichier de test étendu
````
```suggestion
### Fichier de test étendu
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+Pour plus d’informations sur la façon de passer des données au backend (en utilisant `httpx` ou `TestClient`), consultez la <a href="https://www.python-httpx.org" class="external-link" target="_blank">documentation HTTPX</a>.
+
+!!! information
+    à otez que le `TestClient` reçoit des données qui peuvent être converties en JSON, et non en modèles Pydantic.
```
```suggestion
    à noter que le `TestClient` reçoit des données qui peuvent être converties en JSON, et non en modèles Pydantic.
```

GDemay commented on docs/fr/docs/tutorial/tests.md:
```diff
@@ -0,0 +1,216 @@
+</div>
+
+
+Cela détectera automatiquement les fichiers et les tests, les exécutera et vous rapportera les résultats.
```
```suggestion
Cela détectera automatiquement les fichiers et les tests, les exécutera, puis vous rapportera les résultats.
```

GDemay reviewed (changes requested):
I improved the translation and fixed the errors. To be transparent, I think there are still some mistakes in grammar, accents, etc., but it's a good improvement.

GDemay commented:
Why does the smokeshow / coverage at 99% ? Can we start a new build? @Mgcsuper

alejsdev committed:
Merge branch 'master' into master

alejsdev committed:
Merge branch 'master' into master

svlandeg committed:
Merge branch 'master' into master
